panelapp/migrations/0003_panelapp_aus_domain_migration.py

```diff
@@ -0,0 +1,20 @@
+from django.db import migrations
+
+
+def update_urls(apps, schema_editor):
+    PaLocusList = apps.get_model('panelapp', 'PaLocusList')
+
+    # Replace the old domain with the new domain in the url field
+    for entry in PaLocusList.objects.filter(url__startswith='https://panelapp.agha.umccr.org'):
+        entry.url = entry.url.replace('https://panelapp.agha.umccr.org', 'https://panelapp-aus.org')
+        entry.save()
```

Comment:
how long does this migration take to run? We tend to use bulk updates for large data migrations like this as our application startup times out after 5 minutes if pending migrations can not complete in that time.
Here is an example migration that has simialr behavior that uses the bulk_update functionality as an example: https://github.com/broadinstitute/seqr/blob/master/seqr/migrations/0068_project_vlm_contact_email.py#L6

Reply:
The DB migration script should only take a few seconds at most.  The table is tiny in our instance, only 310 rows.  For your instance, I'm guessing around 1000 rows which should still take a few seconds only.